core/assets/bundles/bundle.properties

```diff
@@ -944,6 +944,7 @@ block.surge-tower.name = Surge Tower
 block.diode.name = Battery Diode
 block.battery.name = Battery
 block.battery-large.name = Large Battery
+block.surge-capacitor.name = Surge Capacitor
```

Comment:
For the French translation: `block.surge-capacitor.name = Condensateur de Surtension`.